### PR TITLE
Return descriptive error for unknown permissions on API

### LINKED ISSUE #310: 500 error returned when permission does not exist
We should return 400 Bad request response.

Failing unit test: https://github.com/genialis/resolwe/pull/309

### PR DESCRIPTION
**Pull request by dblenkus**
Closes #310.

### CODE REVIEW

File: docs/CHANGELOG.rst [hunk 1]
```diff
@@ -13,12 +13,18 @@ Changed
...
-- **BACKWARD INCOMPATIBILE:** Rename ``permissions`` attribute on API 
+- **BACKWARD INCOMPATIBILE:** Rename ``permissions`` attribute on API
   endpoints to ``current_user_permissions``
 - API ``permissions`` endpoint raises error if no owner is assigned to
   the object after applied changes
 - ``owner`` permission cannot be assigned to a group
 
+Fixed
+-----
+- Return more descriptive error if non-existing permission is send to
```
Comment (kostko) [inline docs/CHANGELOG.rst]: send -> sent

File: resolwe/permissions/utils.py [hunk 2]
```diff
@@ -101,6 +102,15 @@ def update_permission(obj, data):
     content_type = ContentType.objects.get_for_model(obj)
     full_permissions = list(zip(*obj._meta.permissions))[0]  # pylint: disable=protected-access
 
+    def apply_perm(perms, entity, perm_func):
```
Comment (kostko) [inline resolwe/permissions/utils.py]: Add documentation block explaining what the function does and what parameters it should be given.

Comment (dblenkus) [inline resolwe/permissions/utils.py]: Done.

**Comment by kostko** [comment]
What's up with tests?

**Comment by dblenkus** [comment]
I'm trying to figure out, they pass locally.. :)

**Pull request merged**